### ISSUE TITLE
JS CODE STYLE deprecated

### ISSUE THREAD

**Issue by bichotll** [braziljs/js-the-right-way#274]
Hi,
It currently claims that the best tool is JSHint. JSCS has recently merged ESlint. 
I think ESLint is actually the best and most used tool for "linting" as well as for code style.

**Comment by allanesquina** [comment]
I think we can change it, but the idea is show all the great tools we have. I'm working on a reorganization task, so I'm gonna create a voting issue with the best tools we have today, what do you think? So we can update the list and create new sections if necessary.

**Comment by bichotll** [comment]
I think that a voting is cool ;) That will definitely help to update it.

JS world is updated/changes every day. That could be done every X time (6
months?).
This is a popular page, so we should keep it up to day.

**Comment by allanesquina** [comment]
That's true, we have new features almost every day, and it's a bit hard to keep all things up to date. I have no idea how we can make it here, since there's so many content to list and sounds a bit noise put all of those in a single issue, confusing I guess, I was think in a second options to do that which is to put a button UP/DOWN for each link on the page, so the user can rating the link, but it needs a server to the API at least.
We can split the issues by section as well, which would be more clear
Any another idea?

Thanks man :)

**Comment by bichotll** [comment]
I like the votes idea! But yeah, that needs to be implemented with a
server. Also, for any new tool/resource added it will take a while until is
has a good position.

Maybe ordering the tools automatically by their Github stars? That does not
include books or websites tho...
Then we invite the readers to suggest any tool with a simple button that
opens an issue? On the right of the page?

I would go for basic and easy information overall anyway.

I'm currently quite busy, but I'll do my best ;)

On 24 June 2016 at 22:34, Allan Esquina <notifications@github.com> wrote:

> That's true, we have new features almost every day, and it's a bit hard to
> keep all things up to date. I have no idea how we can make it here, since
> there's so many content to list and sounds a bit noise put all of those in
> a single issue, confusing I guess, I was think in a second options to do
> that which is to put a button UP/DOWN for each link on the page, so the
> user can rating the link, but it needs a server to the API at least.
> We can split the issues by section as well, which would be more clear
> Any another idea?
>
> Thanks man :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/braziljs/js-the-right-way/issues/274#issuecomment-228467610>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAnRHyNSZp3QpjWxHYMvi6CCt-7ETIn5ks5qPE1xgaJpZM4I7lWA>
> .
>



-- 
jaume@bic.cat
630 77 555 0

<http://bic.cat>
*complex but simple*

**Comment by allanesquina** [comment]
Yeah man, I completely understand, we all are too busy, no worries. 
The idea to sort by Github stars is amazing, I like that. We can think about this button, maybe something to incentive people to participate.
I have to finish a project before start to do this changes, so, we have time to think more about it yet. You're welcome here, feel free to contribute always. 
Thanks man :+1:
